README.md

```diff
@@ -44,65 +45,106 @@ Usage instructions are available in the bootstrap [README](./0-bootstrap/README.
 
 ### [1. org](./1-org/)
 
-The purpose of this stage is to set up top level folders used to house projects which contain shared resources such as monitoring, networking, org level logging and also to set baseline security settings through organizational policy.
+The purpose of this stage is to set up the common folder used to house projects which contain shared resources such as DNS Hub, Interconnect, SCC Notification, org level secrets and org level logging.
 This will create the following folder & project structure:
 
 ```
 example-organization
 └── common
-    ├── logs
-    │   ├── org-audit-logs
-    │   └── org-billing-logs
-    ├── monitoring
-    │   ├── org-monitoring-nonprod
-    │   └── org-monitoring-prod
-    └── networking
-        ├── org-shared-vpc-nonprod
-        └── org-shared-vpc-prod
+    ├── prj-p-org-audit-logs
+    ├── prj-p-org-billing-logs
+    ├── prj-p-org-dns-hub
+    ├── prj-p-org-interconnect
+    ├── prj-p-org-scc
+    └── prj-p-org-secrets
 ```
 
 #### Logs
 
-Under the logs folder, two projects are created. One for organization wide audit logs and another for billing logs.
+Under the common folder, two projects are created. One for organization wide audit logs and another for billing logs.
```

Comment:
nit: I would slightly reword this something like `Among the four projects created under the common folder, two projects (prj-p-org-audit-logs,prj-p-org-billing-logs) are used for logging.`

Reply:
> nit: I would slightly reword this something like `Among the four projects created under the common folder, two projects (prj-p-org-audit-logs,prj-p-org-billing-logs) are used for logging.`

Agreed!